packages/core/src/server/prodServer.ts

```diff
@@ -165,8 +165,10 @@ export async function startProdServer(
     {
       pwd: context.rootPath,
       output: {
-        path: config.output.distPath.root || ROOT_DIST_DIR,
-        assetPrefix: config.output.assetPrefix,
+        path: context.distPath,
+        assetPrefixes: Object.values(
+          context.normalizedConfig?.environments || [],
```

Comment:
Can we use `context.environments` here

Reply:
`[]` should be `{}`

Reply:
`output.assetPrefix` is not in `context.environments`, and it's not commonly used.

Reply:
Why `output.assetPrefix` is not in `context.environments`, I think these two code are basically the same.

```js
const assetPrefixes  = Object.values(context.environments).map(
  (item) => pathnameParse(item.config.output.assetPrefix),
);
```

```js
const assetPrefixes = Object.values(
  context.normalizedConfig?.environments || {},
).map((e) => pathnameParse(e.output.assetPrefix));
```